src/reviewers/formPage1.php

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
-<?php session_start();
-include_once 'creds.php';
+<?php
 require_once '../includes/init.php';
+include_once 'creds.php';
```

Comment:
Are they even using the database on this page?

Reply:
looks like it's not. I just removed the creds completely instead of moving it down